src/components/Calculator.jsx

```diff
@@ -1,101 +1,46 @@
 /* eslint-disable */
```

Comment:
- [ ] Kindly avoid disabling linters as it is not a good practice, it is better for your linters to pass the linter checks 👍🏿